packages/toolkit/.changes/next-release/Feature-48c399ea-d42f-4ea1-8f32-d8bc1486a107.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Support starting CloudWatch Logs Live Tail sessions for a specific Lambda function's log group, without having to find which log group manually, but just starting from the function directly."
```

Comment:
suggestion:

```suggestion
	"description": "From the Lambda treeview in AWS Explorer, you can now right-click on a function name and start a CloudWatch Logs Live Tail sessions for the selected function."
```

Reply:
It does look clearer. I updated with this suggestion.